### PR TITLE
Update supl-csp.md

### PR DESCRIPTION
**Pull request by Cheoljin**
In Vibranium, Location team has added FullVersion field (which is extension/generalization of existing Version field. Thus, we add the description for FullVersion. Also, to be clear, we update the description of existing version field. SUPL DDF should be updated accordingly. I will put a new PR for that.

### CODE REVIEW

File: windows/client-management/mdm/supl-csp.md [hunk 1]
```diff
@@ -86,7 +86,10 @@ If this value is not specified, the device infers the H-SLP address from the IMS
 For OMA DM, if the format for this node is incorrect the entry will be ignored and an error will be returned, but the configuration service provider will continue processing the rest of the parameters.
 
 <a href="" id="version"></a>**Version**  
-Optional. Determines the version of the SUPL protocol to use. For SUPL 1.0, set this value to `1`. For SUPL 2.0, set this value to `2`. The default is 1.
+Optional. Determines which SUPL major version the GPS driver should use. (Refer to FullVersion field for extended usage). For SUPL 1.0.0, set this value to `1`. For SUPL 2.0.0, set this value to `2`. The default is 1.
+
+<a href="" id="version"></a>**FullVersion**  
+Optional. Determines which full version, X.Y.Z that the GPS driver should use. X, Y, and Z are major, minor, and service indicator version, respectively. If FullVersion field is defined, Version field that defines only major version is ignored.
```
Comment (illfated) [inline windows/client-management/mdm/supl-csp.md]: Cosmetic grammatical details, but it improves readability and immediate understanding.
```suggestion
Optional. Determines which full version, X.Y.Z that the GPS driver should use. X, Y, and Z are the major, minor, and service indicator versions, respectively. If FullVersion field is defined, a Version field that defines only the major version is ignored.
```

File: windows/client-management/mdm/supl-csp.md [hunk 2]
```diff
@@ -86,7 +86,10 @@ If this value is not specified, the device infers the H-SLP address from the IMS
 For OMA DM, if the format for this node is incorrect the entry will be ignored and an error will be returned, but the configuration service provider will continue processing the rest of the parameters.
 
 <a href="" id="version"></a>**Version**  
-Optional. Determines the version of the SUPL protocol to use. For SUPL 1.0, set this value to `1`. For SUPL 2.0, set this value to `2`. The default is 1.
+Optional. Determines which SUPL major version the GPS driver should use. (Refer to FullVersion field for extended usage). For SUPL 1.0.0, set this value to `1`. For SUPL 2.0.0, set this value to `2`. The default is 1.
```
Comment (illfated) [inline windows/client-management/mdm/supl-csp.md]: Grammatical detail, but it improves readability and immediate understanding.
```suggestion
Optional. Determines which SUPL major version the GPS driver should use. (Refer to the FullVersion field for extended usage). For SUPL 1.0.0, set this value to `1`. For SUPL 2.0.0, set this value to `2`. The default is 1.
```

**Comment by Cheoljin** [comment]
Out of curiosity, in the (embedded) description itself, should we mention that this change (feasibility of FullVersion) is applicable only from Vb? I assume that we need to indicate that with a way.

**Comment by illfated** [comment]
Thank you for the addition of my suggested edits,
Your suggestion looks good to me, but it needs confirmation from either the MS Docs team or someone who knows this topic better than me.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/client-management/mdm/supl-csp.md [hunk 3]
```diff
@@ -86,7 +86,10 @@ If this value is not specified, the device infers the H-SLP address from the IMS
 For OMA DM, if the format for this node is incorrect the entry will be ignored and an error will be returned, but the configuration service provider will continue processing the rest of the parameters.
 
 <a href="" id="version"></a>**Version**  
-Optional. Determines the version of the SUPL protocol to use. For SUPL 1.0, set this value to `1`. For SUPL 2.0, set this value to `2`. The default is 1.
+Optional. Determines which SUPL major version the GPS driver should use. (Refer to the FullVersion field for extended usage.) For SUPL 1.0.0, set this value to `1`. For SUPL 2.0.0, set this value to `2`. The default is 1.
+
+<a href="" id="version"></a>**FullVersion**  
+Optional. Determines which full version, X.Y.Z that the GPS driver should use. X, Y, and Z are the major, minor, and service indicator versions, respectively. If FullVersion field is defined, a Version field that defines only the major version is ignored.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/supl-csp.md]:
```suggestion
Optional. Determines which full version, X,Y, or Z the GPS driver should use. X, Y, and Z are the major, minor, and service indicator versions, respectively. If FullVersion field is defined, a Version field that defines only the major version is ignored.
```

Comment (illfated) [reply windows/client-management/mdm/supl-csp.md]: Suggestion to @JohanFreelancer9 : edit your suggestion to include a space between X, and Y like this:
`Optional. Determines which full version, X, Y, or Z`

Comment (JohanFreelancer9) [reply windows/client-management/mdm/supl-csp.md]: @illfated Thanks followed your suggestion. Changed "or" to "and". Although not obvious here, there is actually a space between X and Y.

Comment (illfated) [reply windows/client-management/mdm/supl-csp.md]: Thank you for your reply and follow-up. A bit odd, though, that the Github diff view does not highlight the space you are seeing.

Comment (illfated) [reply windows/client-management/mdm/supl-csp.md]: Hmm. I just noticed that you suggested to change `X.Y.Z` to `X,Y, and Z` -- which may not be correct in terms of pointing to how the version numbers are represented.
Would it be fair to keep the usage of `X.Y.Z` as a representation of the version number?

Comment (JohanFreelancer9) [reply windows/client-management/mdm/supl-csp.md]: Absolutely. Will edit my suggestion.

Comment (illfated) [reply windows/client-management/mdm/supl-csp.md]: Just out of curiosity (and I may be mistaken), would it be correct to add another comma after the version X.Y.Z to point out that the version representation is a joint clause?
E.g. `Determines which full version, X.Y.Z, the GPS driver should use.`

**Comment by Cheoljin** [comment]
> Thank you for the addition of my suggested edits,
> Your suggestion looks good to me, but it needs confirmation from either the MS Docs team or someone who knows this topic better than me.

Do you know how to get review from the MS Docs team in general? Do you know their IDs to refer (or should I contact them internally?)

**Comment by illfated** [comment]
Well, from my experience in this issue tracker, @e0i (the current handler & member of the MS Docs team following up on this) will make sure to ask the author/manager of the document to review this PR and to answer any questions related to suggested & not yet implemented changes.

**Comment by e0i** [comment]
@manikadhiman The PR that has been contributed by an internal MS member is available for review. Thanks.

**Comment by e0i** [comment]
@Cheoljin The reviews are handled by independent project coordinators by contacting to copy editors and authors, and escalating to other authors or managers where necessary. This process may not be the case for all Docs repositories.

**Comment by DaniHalfin** [comment]
The default branch for this repository has changed. Commenting to remember to merge this to the new default, if this PR gets merged.

**Comment by e0i** [comment]
@Cheoljin 

The default branch for this repository has changed from `master` to `public`. Please edit your pull request to change the merge target to `public` via the `Edit` button at the top right corner. Thanks.

**Pull request closed without merging**